fuzz-rust/Dockerfile

```diff
@@ -1,10 +1,17 @@
 FROM rustlang/rust:nightly
 
 RUN apt-get update && apt-get upgrade -y && \
-    apt-get install -y build-essential clang-11 llvm-11-tools cmake clang curl binutils-dev libbfd-dev libunwind8-dev libblocksruntime-dev pkg-config libudev-dev libssl-dev && \
+    apt-get install -y build-essential clang-11 llvm-11-tools cmake clang curl binutils-dev libbfd-dev libunwind8-dev libblocksruntime-dev pkg-config libudev-dev libssl-dev libgc-dev libtool-bin autoconf && \
     ln -s /usr/bin/llvm-config-11 /usr/bin/llvm-config
 
+RUN wget http://www.hboehm.info/gc/gc_source/gc-8.0.4.tar.gz \
+	&& tar xzf gc-8.0.4.tar.gz \
+	&& cd gc-8.0.4 \
+	&& ./configure \
+	&& make \
+	&& make install
+
```

Comment:
2 things:
- Spacing is tabs, but the rest of the file is spaces, please fix to match
- Isn't gc covered by libgc-dev, a dependency you added?

Reply:
Thanks for verifying that, I'll make sure everything builds as expected, and then I'll merge and update the package immediately.